flink-table-store-core/src/main/java/org/apache/flink/table/store/file/compact/NoopCompactManager.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.store.file.compact;
+
+import org.apache.flink.table.store.file.data.DataFileMeta;
+
+import java.util.Optional;
+import java.util.concurrent.ExecutorService;
+
+/** A {@link CompactManager} which doesn't do things. */
+public class NoopCompactManager extends CompactManager {
```

Comment:
```suggestion
public class DummyCompactManager extends CompactManager {
```

Reply:
Is it necessary to introduce `NoopCompactManager`?

Reply:
I think Noop is better, dummy may occurs in tests code.
`NoopCompactManager` is to avoid intruding into Writer's code.